async/kafka/group/group.go

```diff
@@ -14,15 +14,13 @@ import (
 
 // Factory definition of a consumer factory.
 type Factory struct {
-	name    string
-	group   string
-	topics  []string
-	brokers []string
-	oo      []kafka.OptionFunc
+	name  string
+	group string
+	oo    []kafka.OptionFunc
 }
 
 // New constructor.
-func New(name, group string, topics, brokers []string, oo ...kafka.OptionFunc) (*Factory, error) {
+func New(name, group string, oo ...kafka.OptionFunc) (*Factory, error) {
```

Comment:
brokers and topics are not optional so we should not remove them. OprionFunc is used to configure mostly optional parameters.

Reply:
Hey @mantzas , 
Thanks for the review. I'll update PR according to this requirement.

Reply:
I've updated the PR. Could you please take a look at it one more time?